doc/references.bib

```diff
@@ -2023,4 +2023,19 @@ @Article{Kappenman2021
   publisher = {Elsevier {BV}},
 }
 
+@article{GENOVESE2002870,
```

Comment:
Can you try to match the format for other entries in this file? Probably `GenoveseEtAl2002`?

Reply:
I agree it'd be nice to stay consistent with the other references here.

Reply:
```suggestion
@article{GenoveseEtAl2002,
```